homework2/js/main.js

```diff
@@ -0,0 +1,44 @@
+//There are steps a - d:
+let myName = 'Anastasia';
+console.log('Name:', myName);
+
+let mySurname = 'Fomina';
+console.log('Surname:', mySurname);
+
+let myCity = 'Odessa';
+console.log('City:', myCity);
+
+let mean = (10 + 5 + 2 + 12) / 4;
+console.log('Average:', mean);
+
+/* There are steps e - i:
+We calculate the area and length of a circle,
+display the wrong variables,
+and create comments in these steps.
+ */
+let radius = 5;
+console.log('Circle radius:', radius);
+
+let square = null;
+console.log('Square:', square);
+
+let length;
+console.log('Length:', length);
+
+const pi = 3.14;
```

Comment:
You can initialize this variable as `const PI`.
Common convention declaration is to use all-uppercase letters.